pages/common/shfmt.md

```diff
@@ -0,0 +1,20 @@
+# shfmt
+
+> A shell parser, formatter and interpreter.
+> More information: <https://pkg.go.dev/mvdan.cc/sh/v3>.
```

Comment:
```suggestion
> More information: <https://pkg.go.dev/mvdan.cc/sh>.
```
I would suggest not specifying a version here, as it might get outdated in future.

Reply:
Thank you very much for your suggestions!